fern/docs.yml

```diff
@@ -74,206 +177,22 @@ navigation:
-            slug: previews
-            path: ./docs/pages/fern-docs/config/previews.mdx
-          - section: Customize branding
-            slug: branding
-            contents:
-              - page: Colors, fonts, and background image
-                path: ./docs/pages/fern-docs/config/branding/colors-fonts-background-image.mdx
-                slug: colors-fonts-background-image
-              - page: Logo and favicon
-                path: ./docs/pages/fern-docs/config/branding/logo-and-favicon.mdx
-              - page: Custom domain
-                path: ./docs/pages/fern-docs/config/branding/custom-domain.mdx
-
-      - section: Write content
-        slug: content
-        contents:
-          - page: Write Markdown content
-            path: ./docs/pages/fern-docs/content/write-markdown.mdx
-            slug: write-markdown
-          - page: Frontmatter
-            path: ./docs/pages/fern-docs/content/front-matter.mdx
-          - page: Code snippets
-            path: ./docs/pages/fern-docs/content/code-snippets.mdx
-      
-      - section: API references
-        contents: 
-          - page: Generate API reference
-            path: ./docs/pages/fern-docs/content/generate-api-ref.mdx
-            slug: generate-api-ref
-          - page: Audiences
-            path: ./docs/pages/fern-docs/config/audiences.mdx
-          - page: SDK snippets
-            path: ./docs/pages/fern-docs/config/sdk-snippets.mdx
-          - page: Endpoint errors
-            path: ./docs/pages/fern-docs/config/endpoint-errors.mdx
-
-      - section: Components
-        slug: components
-        contents:
-          - page: Cards
-            path: ./docs/pages/fern-docs/content/components/card.mdx
-          - page: CodeBlocks
-            path: ./docs/pages/fern-docs/content/components/codeblocks.mdx
-            slug: codeblocks
-          - page: Callout
-            path: ./docs/pages/fern-docs/content/components/callout.mdx
-          - page: Icons
-            path: ./docs/pages/fern-docs/content/components/icons.mdx
-          - page: Availability
-            path: ./docs/pages/fern-docs/content/components/availability.mdx
-          - page: Tabs
-            path: ./docs/pages/fern-docs/content/components/tabs.mdx
-          - page: Accordion
-            path: ./docs/pages/fern-docs/content/components/accordion.mdx
-          - page: Steps
-            path: ./docs/pages/fern-docs/content/components/steps.mdx
-          - page: Endpoint snippets
-            path: ./docs/pages/fern-docs/content/components/endpoint-snippets.mdx
+          - page: fern init
+            path: ./overview/dummy.mdx
+          - page: fern check
+            path: ./overview/dummy.mdx
+          - page: fern generate
+            path: ./overview/dummy.mdx
+          - page: fern login
+            path: ./overview/dummy.mdx
+          - page: fern token
+            path: ./overview/dummy.mdx
+  - api: API Reference
+    api-name: public-api
 
 navbar-links:
```

Comment:
I think we should add a second link for `Get support` which links to https://buildwithfern.com/support 

(Independent project is making that page designed to be more like https://linear.app/contact/support )

Reply:
Should we make that link to something that creates a slack connect channel with us?

Reply:
Love that idea. https://api.slack.com/apis/connect